swingset/src/main/java/com/nqadmin/swingset/utils/SSCommon.java

```diff
@@ -367,17 +381,17 @@ public static Date getSQLDate(final String _strDate) {
 	/**
 	 * parent SwingSet component
 	 */
```

Comment:
@errael I guess the edits for ssComponent may have been a little different than some others that went from "private" to "transient private". Based on your comments on this PR, I think I fell short with the manual merge. If you want to fix/push to WhichRowSet, I can grab from there or you can just reply to this comment with the declaration for ssComponent. Thanks!

Reply:
I placed my comment right above the lines in question making neither line easy to see from the GitHub UI.
```	private final SSComponentInterface ssComponent;
	transient private SSComponentInterface ssComponent = null;
```